src/components/modals/CountryListModal.tsx

```diff
@@ -32,14 +33,11 @@ export const CountryListModal = ({ countryCode = getCountry() ?? 'US', bridge }:
     const logoName = filename ?? name.toLowerCase().replace(' ', '-')
     const source = { uri: `${FLAG_LOGO_URL}/${logoName}.png` }
 
+    // TODO: Update modal built-in margins
     return (
-      <SelectableRow
-        arrowTappable
-        icon={<FastImage source={source} style={styles.image} />}
-        subTitle={alpha}
-        title={name}
-        onPress={() => bridge.resolve(alpha)}
-      />
+      <CardUi4 marginRem={[0.5, 0, 0.5, 0]}>
+        <RowUi4 title={name} body={alpha} icon={<FastImage source={source} style={styles.image} />} onPress={() => bridge.resolve(alpha)} />
```

Comment:
Using RowUi4 changes the design altogether. These rows do not mimic the design of the original Tile which RowUi4 uses. 

The design should have the top Text is `primaryText` color and rem=1 while the second row is smaller with the secondary text color.

Reply:
Reverted, retaining original SelectableRow styling.